bindings/pyroot_experimental/PyROOT/test/array_interface.py

```diff
@@ -45,6 +45,52 @@ def test_STLVector(self):
             self.check_memory_adoption(root_obj, np_obj)
             self.check_shape((2, ), np_obj)
 
+    def test_RTensor(self):
+        shape = ROOT.std.vector("size_t")()
+        shape.push_back(2)
+        for dtype in self.dtypes:
+            root_obj = ROOT.TMVA.Experimental.RTensor(dtype)(shape)
```

Comment:
Maybe not for this PR, but we could think of another Pythonization for RTensor to specify the shape as a tuple.
Actually this pythonization would be reused from many places in ROOT...